liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -1031,11 +1034,9 @@ public boolean include(String fileName,
             if (onUnknownFileFormat == OnUnknownFileFormat.FAIL) {
                 throw e;
             }
-            // This matches only an extension, but filename can be a full path, too. Is it right?
-            boolean matchesFileExtension = StringUtil.trimToEmpty(normalizedFilePath).matches("\\.\\w+$");
-            if (matchesFileExtension || onUnknownFileFormat == OnUnknownFileFormat.WARN) {
+            if (onUnknownFileFormat == OnUnknownFileFormat.WARN) {
                 Scope.getCurrentScope().getLog(getClass()).warning(
-                        "included file " + normalizedFilePath + "/" + normalizedFilePath + " is not a recognized file type"
+                        "included file " + normalizedFilePath + " is not a recognized file type"
                 );
```

Comment:
not logging the OnUnknownFileFormat for hidden files or version control system file.